ChangeLog

```diff
@@ -1,3 +1,75 @@
+* Thu Oct 15 2020 Chris Lumens <clumens@redhat.com> Pacemaker-2.0.5
+- Changesets: 497
+- Diff: 285 files changed, 22704 insertions(+), 14458 deletions(-)
+
+- Features added since Pacemaker-2.0.4
+  + cibsecret: read value to set from input (or stdin) if not specified
+  + controller: add new IPC API command for getting node list
+  + libpe_status: Add type="integer" to rule elements, allowing for
+				  specifying 64-bit integers.
+  + scheduler: Add rule-based tests to op_defaults and rsc_defaults.
+  + scheduler: Add on-fail=demote and no-quorum-policy=demote recovery
+               policies for promoted resources.
+  + tools: Add --resource= to filter crm_mon output for a resource.
+  + tools: Add -P to crmadmin to ping via pacemakerd-api.
+  + tools: Use glib for cmdline handling in crmadmin and crm_resource.
+
+- Fixes for regressions introduced in Pacemaker-2.0.4
+  + scheduler: Add the node name back to bundle instances.
+  + tools: get stonith_admin --list-targets working again
+
+- Fixes for regressions introduced in Pacemaker-2.0.3
+  + scheduler: Fix adding HTTP headers to crm_mon in daemon mode.
+
+- Fixes for regressions introduced in Pacemaker-2.0.1
+  + scheduler: require pre-/post-start notifications correctly.
+
+- Changes since Pacemaker-2.0.4
+  + CIB: use crm_exit to free qb-logging
+  + cibsecret: don't use pssh -q option unless supported
+  + controller: Log correct timeout for timed-out stonith monitor
+  + controller: properly detect remote node info requests
+  + daemons: Accept PCMK_OCF_DEGRADED and PCMK_OCF_DEGRADED_MASTER status codes
+  + daemons: Notify tengine of completion of CRM_OP_LRM_DELETE.(CLBZ:#5424)
+  + daemons: Remove relayed stonith operation.(CLBZ#5401)
+  + executor: only send executor notifications to executor clients
+  + fencer:  Add an additional check to advance_topology_level.
+  + fencer: avoid infinite loop if device is removed during operation
+  + fencer: avoid possible use-of-NULL when parsing metadata
+  + ipc-api: allow calling connect after disconnection
+  + ipc-api: avoid infinite loop when disconnected
```

Comment:
I'm pretty sure the above two "ipc-api" entries were for issues within the release, so no entry needed